docs/_main/getting_started.md

```diff
@@ -22,23 +22,19 @@ status. Before we can call any methods, we need to configure our new app with th
 ## Getting a token to use the Web API
 
 Navigate to "OAuth & Permissions" and scroll down to the section for scopes. Slack describes the various permissions
```

Comment:
Just so it's more glanceable, I'd bold "OAuth & Permissions" instead of putting it in quotes. Same goes for whenever the user is accessing the sidebar. This is typically how we do it throughout the API documentation